examples/platform-switching/rocLovesZig.roc

```diff
@@ -1,3 +1,5 @@
 app [main] { pf: platform "zig-platform/main.roc" }
 
-main = "Roc <3 Zig!\n"
+helper = \a, b -> Str.concat a b
+
+main = helper "Roc <" "3 Zig!\n"
```

Comment:
Is this a change you want to keep? If so, should we make it in all the `platform-switching` examples.

Reply:
no, fixed it. a bit sad that this re-runs all CI but ...